src/components/StatusBadge/StatusBadge.tsx

```diff
@@ -43,10 +43,10 @@ export const StatusBadge: ChakraComponent<
 > = chakra(
   forwardRef<HTMLDivElement, React.PropsWithChildren<StatusBadgeProps>>(
     (props, ref?) => {
-      const { children, id, type, ...rest } = props;
+      const { children, id, variant, ...rest } = props;
```

Comment:
Not sure if this would be out of scope, but the default value for `variant` can be set to `"neutral"`, then the configuration for `useStyleConfig` can be simplified.